src/reference/antora/modules/ROOT/nav.adoc

```diff
@@ -124,6 +124,7 @@
 ** xref:amqp/amqp-1.0.adoc[]
 * xref:camel.adoc[]
 * xref:cassandra.adoc[]
+* xref:cloudevents-transform.adoc[]
```

Comment:
Not a fact that it always be only for transformers.
Please, rename the file to `cloudevents`.